artiq/coredevice/comm_kernel.py

```diff
@@ -651,6 +652,10 @@ def read_exception_string():
             message = read_exception_string()
             params = [self._read_int64() for _ in range(3)]
 
+            # Add channel name
```

Comment:
Obvious from the code, remove

Reply:
A more relevant comment would be to stress out that ``params[1]`` is now reserved for the channel number.

Reply:
And the place for such a comment would be coredevice.exceptions I guess.

Reply:
And why not make it ``params[0]``? It's a bit weird to have unreserved, reserved, unreserved,... instead of reserved, unreserved, ...

Reply:
And please filter by exception type instead of attempting to replace in every exception. There are only a few exceptions where this replacement needs to occur.